static/sass/partials/_header.scss

```diff
@@ -1,9 +1,27 @@
+@import 'colors';
+
+$logo-pc-width: 35%;
```

Comment:
I am really not sure about all the new variables. We will not re-use most of that.